http-server/src/main/java/io/airlift/http/server/testing/TestingHttpServerModule.java

```diff
@@ -54,8 +56,12 @@ protected void setup(Binder binder)
         binder.disableCircularProxies();
 
         configBinder(binder).bindConfig(HttpServerConfig.class);
-        configBinder(binder).bindConfigDefaults(HttpServerConfig.class, config -> config.setHttpPort(httpPort));
+        configBinder(binder).bindConfigDefaults(HttpServerConfig.class, config -> config
+                .setHttpPort(httpPort)
+                .setMinThreads(8)
+                .setMaxThreads(32));
```

Comment:
Testing server can use even fewer threads. Max 4 could be enough imo.
Make this separate commit (or maybe even PR for vis).